### PR TITLE
Just translations and language templates revision update

### PR DESCRIPTION
**Pull request by sysadminmatmoz**

### CODE REVIEW

File: OCA_descriptions/i18n/en.po [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline OCA_descriptions/i18n/en.po]: You can remove this file. English translation is on pot file.

File: OCA_descriptions/i18n/en_US.po [hunk 2]
```diff
@@ -1,19 +1,21 @@
 # Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline OCA_descriptions/i18n/en_US.po]: This file is also not needed (as there are no changes).

File: localizable_project/i18n/en_US.po [hunk 3]
```diff
@@ -1,19 +1,21 @@
 # Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline localizable_project/i18n/en_US.po]: The same

File: project_claims/i18n/en_US.po [hunk 4]
```diff
@@ -1,19 +1,21 @@
 # Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline project_claims/i18n/en_US.po]: The same

Comment (sysadminmatmoz) [inline project_claims/i18n/en_US.po]: I didn't put new translations in, just synced the existing ones with the build... anyway with the new commit the translations (en, en_US) you mentioned were removed and templates put at their place.

**Comment by pedrobaeza** [comment]
Thanks! :+1:

**Pull request merged**